ray-operator/config/manager/cluster-scope/manager.yaml

```diff
@@ -25,19 +25,21 @@ spec:
 #        - --enable-leader-election
         image: kuberay/operator
         name: ray-manager
+        ports:
+        - containerPort: 8080
         securityContext:
           allowPrivilegeEscalation: false
         livenessProbe:
           httpGet:
             path: /metrics
-            port: http
+            port: 8080
```

Comment:
I saw operator error and was not able to run. I saw the issue is due to incorrect port configurations.

Reply:
@Jeffwan

Reply:
@scarlet25151

Reply:
nice catch. em. I am wondering why the CI test can not catch the issue?